public/controllers/login-signup.js

```diff
@@ -85,6 +86,31 @@ controller.controller('Login', ['ConfigService', '$scope', '$http', '$location',
                 alert("login failed, reason: " + err.reason);
             });
         };
+        $scope.resetPassword = function(){
+            var url = ConfigService.serverIp + '/forgetPassword';
+            $http.post(url, {username: $scope.forgottenUsername}).success(function(response) {
+                alert(response);
+            }).error(function (err) {
+                alert("Some Error occured: " + err.reason);
```

Comment:
this is not working when i input an incorrect username. i think thats because you are not using the response object to serve the request but are just returning values at these lines https://github.com/omeryousaf/cbs_community/blob/78d3018b18388ae8742de2e3505bcff14d3dadab/services/forgetPasswordService.js#L13, https://github.com/omeryousaf/cbs_community/blob/78d3018b18388ae8742de2e3505bcff14d3dadab/services/forgetPasswordService.js#L16.